csharp/EmsApi.Client/V2/Service/EmsApiEndpoints.cs

```diff
@@ -0,0 +1,24 @@
+﻿
+namespace EmsApi.Client.V2
+{
+    /// <summary>
+    /// Well-known API endpoints.
+    /// </summary>
+    public static class EmsApiEndpoints
+    {
+        /// <summary>
+        /// The default endpoint.
+        /// </summary>
+        public const string Default = Efoqa;
+
+        /// <summary>
+        /// The primary production endpoint.
+        /// </summary>
+        public const string Efoqa = "https://ems.efoqa.com/api";
+
+        /// <summary>
+        /// The beta endpoint.
+        /// </summary>
+        public const string Beta = "https://emsapibeta.ausdig.com/api";
+    }
+}
```

Comment:
If you wanted to fill this out, there are a few more that could be added:

Nightly build system:
https://emsapitest.ausdig.com

stable system:
https://emsapi.ausdig.com/

We should also target the predix endpoints as well. The routes are almost identical except for slight differences in the way you specify the target EMS system. it's 'qantas' instead of '9' or wahtever.